BoxContentSDK/BoxContentSDKTests/file_all_fields_representations.json

```diff
@@ -0,0 +1,108 @@
+{
+    "type": "file",
+    "id": "123",
+    "etag": "0",
+    "parent": {
+        "type": "folder",
+        "id": "0",
+        "sequence_id": null,
+        "etag": null,
+        "name": "All Files"
+    },
+    "path_collection": {
+        "total_count": 1,
+        "entries": [
+                    {
+                    "type": "folder",
+                    "id": "0",
+                    "sequence_id": null,
+                    "etag": null,
+                    "name": "All Files"
+                    }
+                    ]
+    },
+    "sha1": "f34df557706317b50ee400e072eb731032fe27d3",
+    "name": "Halloweenie-3.jpg",
+    "created_at": "2014-09-22T17:02:22-07:00",
+    "modified_at": "2014-09-22T17:02:23-07:00",
+    "content_created_at": "2014-09-11T18:40:43-07:00",
+    "content_modified_at": "2014-09-11T18:40:44-07:00",
+    "description": "",
+    "size": 622592,
+    "created_by": {
+        "type": "user",
+        "id": "13338532",
+        "name": "Michelle Brown",
+        "login": "booxqa@gmail.com"
+    },
+    "modified_by": {
+        "type": "user",
+        "id": "13338532",
+        "name": "Michelle Brown",
+        "login": "booxqa@gmail.com"
+    },
+    "trashed_at": null,
+    "purged_at": null,
+    "owned_by": {
+        "type": "user",
+        "id": "13338532",
+        "name": "Michelle Brown",
+        "login": "booxqa@gmail.com"
+    },
+    "shared_link": null,
+    "item_status": "active",
+    "comment_count": 0,
+    "permissions": {
+        "can_download": true,
+        "can_preview": true,
+        "can_upload": true,
+        "can_comment": true,
+        "can_rename": true,
+        "can_delete": true,
+        "can_share": true,
+        "can_set_share_access": true
+    },
+    "allowed_shared_link_access_levels": [
+                                          "collaborators",
+                                          "open"
+                                          ],
+    "collections": [
+                    {
+                    "type": "collection",
+                    "id": "10047",
+                    "name": "Favorites",
+                    "collection_type": "favorites"
+                    }
+                    ],
+    "representations": {
+        "entries": [
+        {
```

Comment:
xcode 8.3 seems to have formatting issues, attempted to format by hand and getting different results in git than in xcode. Will leave as is.